ServiceApp/server/package.json

```diff
@@ -30,7 +30,7 @@
         "copyfiles": "^2.2.0",
         "cors": "^2.8.5",
         "express": "^4.17.1",
-        "identity_ts": "iotaledger/identity.ts#910ebdb01496f7a1abf115d9b272ca4456aa00ba",
+        "identity_ts": "LisaSchneider94/identityIMP.ts",
```

Comment:
sorry, but not like this. we don't need a fork, and the IF should be the maintainer of the repo

Reply:
I will create another PR to make IMP compatible with the latest version of the identity.ts.
Please join the PR to add changes and run tests